### ISSUE TITLE
`copilot` - Upon Run Add To History/Clipboard

### ISSUE THREAD

**Issue by mattcargile** [dfinke/PowerShellAI#87]
When receiving code and running it through `copilot`, it would be convenient if we could automatically add to the `PSReadLine` history or history in general. Or it would be a cool feature to add the code to the clipboard.

Could use the below for the `PSReadLine` history. And/Or `Add-History` for the session. `Set-Clipboard` would be what I was thinking for the _clipboard_.
```powershell
[Microsoft.PowerShell.PSConsoleReadLine]::AddToHistory($line)
```

### Workaround
This could all be for naught. I could use the `enable-AIShortCutKey` though the context is thrown off a bit a la #77 .

**Comment by mattcargile** [comment]
Hmm. Maybe it is because you have _Plus_ and are using _GPT-4_? I just tried this and got different results though both are wrong. I ended up having to go to _Google_ for that. :-) 

I know there is some randomization...

![image](https://user-images.githubusercontent.com/74641867/226478048-e471eece-9d5c-4c46-830d-e67f698b7900.png)

**Comment by mattcargile** [comment]
Ooo. GPT4 is going well!

**Comment by mattcargile** [comment]
> Does that response look good?

Yes, that response is much better. The module name appears to be off. On newer versions of Windows like 2016, the module would be `IISAdministration`. It appears that on older versions of Windows `WebAdministration` had a different cmdlet. First time I had researched the difference between `WebAdministration` and `IISAdministration`.

**Comment by dfinke** [comment]
Next step is to flesh out the prompt. A few months back gpt/github copilot would respond about Pester using the older syntax. I would add `use the newest Pester syntax` and that would give the model enough to then respond with v5.x syntax.

**Comment by dfinke** [comment]
Now avaiable.
